src/Apps/Legion/Contexts/DataManagementContext/DataManagementContext.types.ts

```diff
@@ -0,0 +1,47 @@
+import { IADXConnection, IAppData, PIDDocument } from '../../Models/Interfaces';
+
+export interface IDataManagementContext {
+    dataManagementContextState: IDataManagementContextState;
+    dataManagementContextDispatch: React.Dispatch<DataManagementContextAction>;
+}
+
+export interface IDataManagementContextState {
+    // Keep adding source types here
+    sources: Array<IADXConnection | PIDDocument>;
+    initialAssets: IAppData;
```

Comment:
not sure if we need to keep initialAssets and modifiedAssets separately but I think it is fine for now.

Reply:
we only need them right now to be able to compare and know what was changed during this session. So if you come back from a future step we can build out the view model correctly again, like if you unselect a property we know whether to check it. There are other ways to do track that, but this seemed like a decent naive implementation that would extend to other scenarios.

Reply:
we need this to track initial values and selected/unselected values.